### PR TITLE
cockpit.js: translate a couple more error codes

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
cockpit.message() is missing the most obvious error (ENOENT) and another common one that we see leaking through the UI in the new editor component in Cockpit Files.

Add these.

### CODE REVIEW

File: pkg/lib/cockpit.js [hunk 1]
```diff
@@ -2618,6 +2618,10 @@ function factory() {
             return _("Cockpit could not contact the given host.");
         else if (problem == "too-large")
             return _("Too much data");
+        else if (problem == "not-found")
+            return _("No such file or directory");
```
Comment (martinpitt) [inline pkg/lib/cockpit.js:2622]: This is *not* how `not-found` is [defined](https://cockpit-project.org/guide/latest/cockpit-error.html). We use it for D-Bus, PackageKit transactions, etc. This error *really* needs to be handled on the caller side which has all the necessary context to build an useful error mesage.

File: pkg/lib/cockpit.js [hunk 2]
```diff
@@ -2618,6 +2618,10 @@ function factory() {
...
+        else if (problem == "change-conflict")
+            return _("The existing file changed unexpectedly");
```
Comment (martinpitt) [inline pkg/lib/cockpit.js:2624]: This is less bad, but still generic -- i.e. would be better to include the actual path in the message.

**Review by martinpitt — CHANGES_REQUESTED** [review]
Let's please not do this. Please handle this in c-files.

**Comment by martinpitt** [comment]
Or rather: To fix this problem on the lower cockpit.js/bridge level, the throwers need to add a proper `message` field to the exception -- the cockpit.message() will use that. We are apparently missing a few places, but cockpit.message() is the worst possible place to try and divine good error messages.

**Comment by allisonkarlitskaya** [comment]
OK.

**Pull request closed without merging**